cypress/e2e/components/header.cy.js

```diff
@@ -1,42 +1,37 @@
-import selectors from '../constants/selectors';
+import selectors, { componentSelectors } from '../constants/selectors';
 import { routes } from '../constants/url';
 
-Cypress.Commands.add('inHeader', (callback) => {
-    cy.get('header').within(callback);
-});
-
 describe('check headerimage and content', () => {
     beforeEach(() => {
         cy.viewport(1920, 1080);
         cy.visit(routes.home);
     });
     context('check topheader links', () => {
         it('check adress link', () => {
-            cy.inHeader(() => {
-                const adressLink = cy.get(selectors.link);
-                adressLink
+            cy.get(componentSelectors.topHeader).within(() => {
```

Comment:
warum hast du dich gegen den command entschieden?

Reply:
habe mich bewusst dagegen entschieden, da das mein allererster versuch war etwas lauffähiges zu haben und nachher ein anderes schea gefahren bin. dadurch, dass ich die const auch nicht wirklich mehrfach benutze habe ich den custom command verworfen

Reply:
Nachteil ist jetzt, dass wenn sich am selektor vom header etwas ändert, du das in jedem Testfall machen musst. 
Imho ist der custom command `inHeader` da die bessere Wahl, da er hier redundanz entfernt.

Reply:
verstehe ich, aber das nehme ich in dem Fall jetzt so hin mit dieser Variante.